src/platform/web/dom/TimeFormatter.ts

```diff
@@ -75,6 +79,37 @@ export class TimeFormatter implements ITimeFormatter {
             return this.otherYearFormatter.format(date);
         }
     }
+
+    formatDuration(milliseconds: number): string {
+        let days = 0;
+        let hours = 0;
+        let minutes = 0;
+        if (milliseconds > DAYS_IN_MS) {
+            days = Math.floor(milliseconds / DAYS_IN_MS);
+            milliseconds -= days * DAYS_IN_MS;
+        }
+        if (milliseconds > HOURS_IN_MS) {
```

Comment:
Should the `>` be `>=` ?
Currently `formatDuration(25 hours in ms)` gives `1d 0h 60m 0s`